index.test-d.ts

```diff
@@ -118,3 +118,21 @@ expectType<CSPair>(ansiStyles.italic);
 expectType<CSPair>(ansiStyles.reset);
 expectType<CSPair>(ansiStyles.strikethrough);
 expectType<CSPair>(ansiStyles.underline);
+
+// --- ModifierName ---
+expectType<ModifierName>('strikethrough');
+expectError<ModifierName>('delete');
+
+// --- ForegroundColorName ---
+expectType<ForegroundColorName>('blue');
+expectError<ForegroundColorName>('pink');
+
+// --- ForegroundColorName ---
+expectType<BackgroundColorName>('bgBlue');
+expectError<BackgroundColorName>('bgPink');
+
+// --- ColorName ---
+expectType<ColorName>('blue');
+expectType<ColorName>('bgBlue');
+expectError<ColorName>('pink');
+expectError<ColorName>('bgPink');
```

Comment:
I'm trying to add some tests for those types.

But I got these errors below:

```
index.test-d.ts:123:0
✖  123:0  Parameter type keyof Modifier is declared too wide for argument type "strikethrough".  
✖  127:0  Parameter type keyof ForegroundColor is declared too wide for argument type "blue".    
✖  131:0  Parameter type keyof BackgroundColor is declared too wide for argument type "bgBlue".  
✖  135:0  Parameter type ColorName is declared too wide for argument type "blue".                
✖  136:0  Parameter type ColorName is declared too wide for argument type "bgBlue". ```

Any suggestions about this part?

Reply:
Use `expectAssignable`